trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/mapFacilityType.ts

```diff
@@ -0,0 +1,54 @@
+import { Facility, FacilityType, FACILITY_TYPE, MAPPED_FACILITY_TYPE } from '@ukef/dtfs2-common';
+
+const { BOND, LOAN, CASH, CONTINGENT } = FACILITY_TYPE;
+
+/**
+ * Maps a facility type to a differently worded facility type, to ultimately be rendered in the UI, as per design.
+ * @param type - the facility type in the database
+ * @param facilitySnapshot - the facility snapshot from the database
+ * @returns the mapped facility type
+ *
+ * @example
+ * // returns 'Bid Bond'
+ * mapFacilityType('Bond', {...facilitySnapshot, code: 'BSS', bondType: 'Bid Bond'})
+ *
+ * @example
+ * // returns 'Loan'
+ * mapFacilityType('Loan', facilitySnapshot)
+ */
+export const mapFacilityType = (type: FacilityType, facilitySnapshot: Facility): string | null => {
```

Comment:
Do we want an explicit `mapBssEwcsFacilityType` name?

Reply:
Good idea!